test/e2e/network/dns_common.go

```diff
@@ -416,19 +424,24 @@ func createProbeCommand(namesToResolve []string, hostEntries []string, ptrLookup
 	for _, name := range hostEntries {
 		fileName := fmt.Sprintf("%s_hosts@%s", fileNamePrefix, name)
 		fileNames = append(fileNames, fileName)
-		probeCmd += fmt.Sprintf(`test -n "$$(getent hosts %s)" && echo OK > /results/%s;`, name, fileName)
+		probeCmd += fmt.Sprintf(`test -n "$$(getent %s %s)" && echo OK > /results/%s;`, getentdb, name, fileName)
+	}
+
+	podARecByUDPFileName := fmt.Sprintf("%s_udp@Pod%sRecord", fileNamePrefix, typeA)
```

Comment:
Since kube-dns does not support this syntax for IPv6, should the block L430-441 be conditional? Or should probes that are not supported by kube-dns be moved to a separate test case and labeled, so that they can be skipped for kube-dns, but allowed for other DNS implementations?

There may be other probes that are not supported by kube-dns...

Reply:
Has this been resolved? Because I agree it should be possible for a cluster to be Conformant without using a specific DNS implementation (CoreDNS vs. kube-dns).  Given that this function ends up being used in conformance tests I view this as a non-starter